<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry"><rect><x>0</x><y>0</y><width>920</width><height>601</height></rect></property>
  <property name="windowTitle"><string>MainWindow</string></property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item><widget class="QPushButton" name="btnRunCuda"><property name="text"><string>Run CUDA</string></property></widget></item>
    <item><widget class="QWidget" name="raylibPlaceholder" native="true"/></item>
    <item>
     <widget class="QWidget" name="renderWidget">
      <property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Expanding"/></property>
      <property name="minimumSize"><size><width>100</width><height>100</height></size></property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="controlPanel">
      <property name="sizePolicy"><sizepolicy hsizetype="Fixed" vsizetype="Expanding"/></property>
      <property name="minimumSize"><size><width>300</width><height>0</height></size></property>
      <property name="maximumSize"><size><width>300</width><height>16777215</height></size></property>
      <layout class="QVBoxLayout" name="verticalLayoutControl">
       <item>
        <widget class="QGroupBox" name="groupSimulation">
         <property name="title"><string>Simulation</string></property>
         <layout class="QFormLayout" name="formLayoutSimulation">
          <item row="0" column="0"><widget class="QLabel" name="labelParticles"><property name="text"><string>Particles</string></property></widget></item>
          <item row="0" column="1"><widget class="QSpinBox" name="spinParticles"/></item>
          <item row="1" column="0"><widget class="QLabel" name="labelRmin"><property name="text"><string>R min</string></property></widget></item>
          <item row="1" column="1"><widget class="QSlider" name="sliderRmin"><property name="orientation"><enum>Qt::Horizontal</enum></property></widget></item>
          <item row="2" column="0"><widget class="QLabel" name="labelRmax"><property name="text"><string>R max</string></property></widget></item>
          <item row="2" column="1"><widget class="QSlider" name="sliderRmax"><property name="orientation"><enum>Qt::Horizontal</enum></property></widget></item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPhysics">
         <property name="title"><string>Physique</string></property>
         <layout class="QFormLayout" name="formLayoutPhysics">
          <item row="0" column="0"><widget class="QLabel" name="labelVmin"><property name="text"><string>V min</string></property></widget></item>
          <item row="0" column="1"><widget class="QSlider" name="sliderVmin"><property name="orientation"><enum>Qt::Horizontal</enum></property></widget></item>
          <item row="1" column="0"><widget class="QLabel" name="labelVmax"><property name="text"><string>V max</string></property></widget></item>
          <item row="1" column="1"><widget class="QSlider" name="sliderVmax"><property name="orientation"><enum>Qt::Horizontal</enum></property></widget></item>
          <item row="2" column="0"><widget class="QLabel" name="labelElasticity"><property name="text"><string>Elasticity</string></property></widget></item>
          <item row="2" column="1"><widget class="QDoubleSpinBox" name="spinElasticity"/></item>
          <item row="3" column="0"><widget class="QLabel" name="labelFriction"><property name="text"><string>Friction</string></property></widget></item>
          <item row="3" column="1"><widget class="QDoubleSpinBox" name="spinFriction"/></item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry"><rect><x>0</x><y>0</y><width>920</width><height>21</height></rect></property>
   <widget class="QMenu" name="menuFichier"><property name="title"><string>Fichier</string></property></widget>
   <widget class="QMenu" name="menuView"><property name="title"><string>View</string></property></widget>
   <widget class="QMenu" name="menuOptions"><property name="title"><string>Options</string></property></widget>
   <addaction name="menuFichier"/><addaction name="menuView"/><addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea"><enum>TopToolBarArea</enum></attribute>
   <attribute name="toolBarBreak"><bool>false</bool></attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/><connections/>
 <slots><slot>on_btnRunCuda_clicked()</slot></slots>
</ui>
